src/gui/src/CapriceOptions.cpp

```diff
@@ -247,6 +247,7 @@ CapriceOptions::CapriceOptions(const CRect& WindowRect, CWindow* pParent, CFontE
     m_pDropDownCPCLanguage->SelectItem(CPC.keyboard);
     m_pDropDownCPCLanguage->SetIsFocusable(true);
     // option 'kbd_layout' which is the platform keyboard layout (i.e. the PC keyboard layout)
+    /*
```

Comment:
Done - we build the dropdown list by listing all the .map files.
The is still the possibility of a race condition if a map file is added to the resources directory after the  GUI is displayed - but I do not really know how to solve this simply.